source/dub/dub.d

```diff
@@ -181,18 +213,32 @@ class Dub {
 		determineDefaultCompiler();
 	}
 
+	Path toPath(PlacementLocation location) const
+	{
+		final switch (location) {
+			case PlacementLocation.local: return m_rootPath;
+			case PlacementLocation.user: return m_dirs.userRepo;
+			case PlacementLocation.system: return m_dirs.systemRepo;
+		}
+	}
```

Comment:
useful for transitioning to using paths when you have `PlacementLocation` all over your code, lets people translate from `PlacementLocation`. If all the `PlacementLocation` overloads were to be removed, I would imagine this would remain to allow stragglers to stay on board.